Automattic/jetpack
# AI Extension: Add keyboard shortcut for stop action on forms and update text

Fixes #31469

## Proposed changes:

* Changes the `Send` text to `Send (ESC)` to make the shortcut visible
* Adds the shortcut to the AIControl component, making it work for Forms as well

### Other information:


## Jetpack product discussion


p1695400366663969-slack-C054LN8RNVA

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block and send a request
* Check that the `ESC` shortcut is visible
* Press ESC
* The suggestion should stop and the focus should be at the input
* Add a Form
* Check that the shortcut is visible and works as well
* Check that there is no regression

The component story has been updated as well and the `Stop` action can be seen on storybook by:
* Running  `( cd projects/js-packages/storybook && pnpm run storybook:dev --no-open )`
* Visiting http://localhost:50240/?path=/story/js-packages-ai-client-ai-control--default&args=state:requesting

| Before | After |
| - | - |
| ![2023-09-22_14-42-45-before](https://github.com/Automattic/jetpack/assets/8486249/93698b93-a9f5-44ba-a348-c05ddd8adc25) | ![2023-09-22_14-41-26-after](https://github.com/Automattic/jetpack/assets/8486249/31185e45-2143-43cf-bcab-960ae087cf81) |
